frontend/src/bundles/create-avatar/components/instruction/instruction.module.css

```diff
@@ -0,0 +1,50 @@
+.flexContainer {
+    flex-direction: column;
+    background-color: white;
+    padding: 30px;
+    border-radius: 10px;
+    align-items: center;
+    gap: 50px;
+    padding-bottom: 80px;
+}
+
+.innerFlex {
+    color: var(--chakra-colors-typography-600);
+    font-size: 14px;
+    justify-content: center;
+    gap: 40px;
+}
+
+.iconContainer {
+    display: flex;
+    gap: 5px;
+    margin-bottom: 30px;
+}
+
+#iconScreenshots {
+    border-radius: 50%;
+    background-color: white;
+    color: var(--chakra-colors-brand-secondary-50);
+}
+
+#iconAvoid {
+    border-radius: 50%;
+    background-color: white;
+    color: var(--chakra-colors-brand-secondary-900);
+}
```

Comment:
Why do we need ids here? Class styles get overridden?

Reply:
In icons chakra theme colors are not showing if we I className but they do if I use id.

Reply:
pull next branch with fix and try to use className